gradle/dependencies.gradle

```diff
@@ -2,22 +2,23 @@ ext {
 
     androidVersions = [
             minSdkVersion    : 14,
-            targetSdkVersion : 25,
-            compileSdkVersion: 25,
+            targetSdkVersion : 26,
+            compileSdkVersion: 26,
             buildToolsVersion: '27.0.2'
     ]
 
     version = [
-            mapboxServices   : '2.2.6',
-            gmsLocation      : '11.0.4',
-            junit            : '4.12',
-            supportLibVersion: '25.4.0',
-            constraintLayout : '1.0.2',
-            mockito          : '2.11.0',
-            testRunnerVersion: '1.0.1',
-            okhttp3          : '3.10.0',
-            gson             : '2.8.2',
-            espressoVersion  : '3.0.1'
+            mapboxServices      : '2.2.6',
+            gmsLocation         : '11.0.4',
+            junit               : '4.12',
+            supportLibVersion   : '26.1.0',
+            constraintLayout    : '1.0.2',
+            mockito             : '2.11.0',
+            testRunnerVersion   : '1.0.1',
+            okhttp3             : '3.10.0',
+            gson                : '2.8.2',
+            espressoVersion     : '3.0.1',
+            archLifecycleVersion: "1.0.0"
```

Comment:
Would there be any benefits to bumping arch components version? I think `1.1.1` requires support lib 27+, but maybe we could use `1.1.0`?

Reply:
Nope 1.1.1 is good with `supportLib` of 26.1.0